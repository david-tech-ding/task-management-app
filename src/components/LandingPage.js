import React from "react";

const LandingPage = () => {
  return (
    <div>
      <h2>Welcome to Clinic Task Manager!</h2>
      <p>
        To continue, please log in to your account or create a new one to get
        started.
      </p>
    </div>
  );
};

export default LandingPage;
