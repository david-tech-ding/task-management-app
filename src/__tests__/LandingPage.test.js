import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import LandingPage from "../components/LandingPage";

describe("renders", () => {
  it("renders correctly", () => {
    const { asFragment } = render(
      <Router>
        <LandingPage />
      </Router>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
