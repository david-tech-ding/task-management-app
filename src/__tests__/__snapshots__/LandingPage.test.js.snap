// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders renders correctly 1`] = `
<DocumentFragment>
  <div>
    <h2>
      Welcome to Clinic Task Manager!
    </h2>
    <p>
      To continue, please log in to your account or create a new one to get started.
    </p>
  </div>
</DocumentFragment>
`;
